app/views/admin/_version.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= turbo_frame_tag dom_id(version) do %>
+  <div class="bg-white m-2 p-2 shadow relative rounded">
+    <% unless version.whodunnit.nil? %>
+      <p><strong><%= t("admin.user") %>:</strong> <%= User.find(version.whodunnit).email %></p>
+    <% end %>
+    <div>
+      <% unless version.item[:name].nil? %>
+        <p><strong><%= t("admin.field") %>:</strong> <%= Space.human_attribute_name(version.item.name) %></p>
```

Comment:
Savner en for Hvilken entity det var for også, typ noe sånt: 

`<h3><%= ENTITY_MODEL =>: <strong><%= ENTITY.title || entity.id %></strong></h3>`

Hvor Model er f.eks. t('space_owner') eller t('space') (eller hvordan nå enn det var igjen for å hente ut oversatte model names)

Title for de modellen som har det, og kun .id for de som ikke har.

Reply:
> Savner en for Hvilken entity det var for også, typ noe sånt:
> 
> `<h3><%= ENTITY_MODEL =>: \<strong\><%= ENTITY.title || entity.id %>\</strong\></h3>`
> 
> Hvor Model er f.eks. t('space_owner') eller t('space') (eller hvordan nå enn det var igjen for å hente ut oversatte model names)
> 
> Title for de modellen som har det, og kun .id for de som ikke har.
Vi kan ganske enklelt gjøre `version.item.class.model_name.human`

Reply:
Ja kan legge inn det